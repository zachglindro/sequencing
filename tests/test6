3 20
2 5 4 1
3 10 5 1
1 20 3 0