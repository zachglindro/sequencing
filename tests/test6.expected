Solution found. Number of S tasks completed: 2
Optimal Schedule: 0 -> 1 -> 2